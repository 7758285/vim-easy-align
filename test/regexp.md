" :source run.vim
" @a

apple;:banana::cake
data;;exchange:;format

