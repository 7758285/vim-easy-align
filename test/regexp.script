4Gvipjyvip:EasyAlign:Pvip:EasyAlign*:Pvip:EasyAlign-:Pvip:EasyAlign * :Pvip:EasyAlign -2 :Pvip:EasyAlign -2:Pvip:easyA€kb€kb€kb€kb€kbEasAli€kb€kb€kbyAlign *:Pvip:EasyAlign;Pvip:EasyAlign /;/Pvip:EasyAlign/;/Pvip:EasyAlign*/;/Pvip:e€kbEasyAlign/[:;]\+/Pvip:EasyAlign*/[:;]\+/Pvip:EasyAlign* /[:;]\+/Pvip:EasyAlign */[:;]\+/Pvip:EasyAlign   *   [:€kb€kb/[:;]\+/   Pvip:EasyAlign*[:;]\+€kb€kb€kb€kb€kb€kb€kb=/[€kb€kb€kb-/[:;]\+/Pvip:EasyAlign-2/[:;\€kb\€kb]\+/Pvip:EasyAlign/[:;]\+/{ 'ml': '<<<', 'mrr': '>>>'}  Pvip:EasyAlign/[:;]\+/{'ml':''m€kb, 'stl': 1}Pvip:EasyAlign * /€kb /[:;]€kb\€kb]\+/ {'stick_to': 1, 'marl': '???', 'marr': '~~~'}  Pvip:EasyAlign /../Pvip:EasyAlign*/../Pvip:EasyAlign ** €kb/[:;]/€kb€kb\+]/Pvip:EasyAlign/[:;\€kb\€kb]\+/'€kb{'marginr':'____'}Pvip:EasyAlign**[€kb/[:;]\+/Pvip:EasyAlignRight**/[:;]\+/Pfbjfm:EasyAlign/[:;]\+/
