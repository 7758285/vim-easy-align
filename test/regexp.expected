" :source run.vim
" @a

apple;:banana  :: cake
data;;exchange :; format

apple ;: banana   ::   cake
 data ;; exchange :; format

apple ;:   banana :: cake
data  ;; exchange :; format

apple ;:____banana::cake
data  ;;____exchange:;format

apple ;  : banana   :  : cake
data  ;  ; exchange :  ; format

ap  pl  e;  :b  an  an  a:  :c  ak e
da  ta  ;;  ex  ch  an  ge  :;  fo  rm  at

ap ple;:banana::cake
da ta;;exchange:;format

apple???;:~~~banana???::~~~  cake
data???;;~~~ exchange???:;~~~format

apple;: banana::cake
data;;  exchange:;format

apple<<<;:>>>banana::cake
data <<<;;>>>exchange:;format

apple ;: banana::cake
data  ;; exchange:;format

apple;:banana  :: cake
data;;exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana   :: cake
data  ;; exchange :; format

apple ;: banana::cake
data  ;; exchange:;format

apple ; :banana::cake
data  ;  ; exchange: ; format

apple ; :banana::cake
data  ; ;exchange:;format

apple ; :banana::cake
data  ; ;exchange:;format

apple;:banana::cake
data;;exchange:;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:banana: :cake
data;;exchange:;format

apple;:banana: :cake
data;;exchange:;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:banana:: cake
data;;exchange: ;format

apple;:         banana: : cake
data;;exchange: ;format

apple;:         banana::cake
data;;exchange: ;format

